src/renderer/components/cluster-manager/clusters-menu.tsx

```diff
@@ -149,14 +144,25 @@ export class ClustersMenu extends React.Component<Props> {
             </Droppable>
           </DragDropContext>
         </div>
-        <div className="add-cluster">
-          <Tooltip targetId="add-cluster-icon">
-            Add Cluster
-          </Tooltip>
-          <Icon big material="add" id="add-cluster-icon" disabled={workspace.isManaged} onClick={this.addCluster}/>
-          {newContexts.size > 0 && (
-            <Badge className="counter" label={newContexts.size} tooltip="new"/>
-          )}
+
+        <div className="WorkspaceMenu">
+          <Icon big material="menu" id="workspace-menu-icon" data-test-id="workspace-menu" />
+          <Menu
+            usePortal
+            htmlFor="workspace-menu-icon"
+            className="WorkspaceMenu"
+            isOpen={this.workspaceMenuVisible}
+            open={() => this.workspaceMenuVisible = true}
+            close={() => this.workspaceMenuVisible = false}
+            toggleEvent="click"
+          >
+            <MenuItem onClick={() => navigate(addClusterURL())} data-test-id="add-cluster-menu-item">
+              <Icon small material="add" /> Add Cluster
+            </MenuItem>
+            <MenuItem onClick={() => navigate(landingURL())} data-test-id="workspace-overview-menu-item">
+              <Icon small material="dashboard" /> Workspace Overview
+            </MenuItem>
+          </Menu>
```

Comment:
this stays visible when user clicks on nav sidebar/dashboard, even after clicking "open cluster settings" icon in top right corner. I haven't figured out how to fix it.